1. A booking manager-ben elleőrizni akartad, hogy egy épp létrehozandó passanger mér létezik-e a DB-ben
2.ehhez kellett  egy passangers = select_passenger(passport_number, table="passengers") hívás, amihez kell egy
3.  def select_passenger(passport_number, table="passengers"): hívás, amihez kell egy
4.     query = get_data("config.ini", "query", "select").format(passport_number, table)

    """create a connection with DB.connection.get_connection()"""
    host = get_data("config.ini", "db", "host")
    port= get_data("config.ini", "db", "port")
    user = get_data("config.ini", "db", "user")
    password = get_data("config.ini", "db", "password")
    connection = get_connection(host, port, user, password)

    De:
    ...
    host = get_data("config.ini", "db", "host")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "C:\Users\A87484215\intermediatePythonKnowledge\AirplaneBookingSystem\utils\utils.py", line 12, in get_data
    raise SectionNotFound(f"The section '{section}' was not found in the config.ini!")
    my_exceptions.exceptions.SectionNotFound: The section 'db' was not found in the config.ini!
